Scripts/Common/Optimize.sh

```diff
@@ -65,7 +65,8 @@ sed -i 's/zramsize=.*/zramsize=50%,max_comp_streams=8/' essential/mata/rootdir/e
 sed -i 's/zramsize=.*/zramsize=50%,max_comp_streams=4/' zte/axon7/rootdir/etc/fstab.qcom &>/dev/null || true;
 sed -i 's/zramsize=.*/zramsize=50%,max_comp_streams=8/' sony/tama-common/rootdir/etc/fstab.qcom &>/dev/null || true;
 #4GB+ (2GB+)
-sed -i 's/zramsize=.*/zramsize=50%,max_comp_streams=8/' google/redbull/fstab.hardware google/sunfish/fstab.hardware oneplus/msm8998-common/rootdir/etc/fstab.qcom oneplus/sdm845-common/rootdir/etc/fstab.qcom xiaomi/sm6150-common/rootdir/etc/fstab*.qcom &>/dev/null || true;
+sed -i 's/zramsize=.*/zramsize=50%,max_comp_streams=8/' google/redbull/fstab.hardware google/sunfish/fstab.hardware oneplus/msm8998-common/rootdir/etc/fstab.qcom oneplus/sdm845-common/rootdir/etc/fstab.qcom &>/dev/null || true;
+sed -i 's/zramsize=.*/zramsize=50%,max_comp_streams=8,zram_backingdev_size=512M/' xiaomi/sm6150-common/rootdir/etc/fstab.zram &>/dev/null || true;
```

Comment:
why did you remove this?

Reply:
I'm just respecting the device tree changes, zram_backingdev_size seems to enable zram writeback with a 512M limit.

Alternatively, the sed regex could be changed to keep additional changes and only modify the values of zramsize and max_comp_streams if that's the intent of the patch.